jest.config.js

```diff
@@ -7,6 +7,10 @@ const commonConfig = {
   snapshotSerializers: ["jest-serializer-path"],
   testEnvironment: "node",
   testPathIgnorePatterns: ["utils.ts"],
+  transformIgnorePatterns: [
+    // These are ESM modules that need to be transpiled before Jest can run them
```

Comment:
do you mean the opposite? these are esm modules that *should not be transpiled*

Reply:
@kevinslin Because `jest` transforms test codes to `commonjs`, the transpiled test code was having trouble in importing esm module libraries. That's why we decided to let `jest` transform esm module libraries to `commonjs` libraries so that they can be imported using `require`.

Because `jest` by default ignores all `node_modules`, we reset the pattern to ignore everything except `esm module` libraries that need to be transpiled.